### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by RodChak**
# [Portfolio milestone  6](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md): Portfolio - mobile menu

### In this milestone, I implemented the following features in mobile:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- Added Scipt.js file Updated README.md file
- Added Scipt.js file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,13 +17,17 @@
...
+                        <li class="nav-item"><a href="#Contact_form" class="nav-link">Contact</a></li>
                     </ul>
+                    <div class="hamburger">
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
                 </nav>
             </header>
             <div class="background_content">
```
Comment (tresorsawasawa) [inline index.html:33]: - [ ] Good job so far, your project is almost completed. I noticed that you miss the `blur effect` on the **Background** of the `menu` to match the Figma design also make sure to add the correct spacing around the `Close_Menu` button **(top && right).**

| ### Figma Design | Your Design |
| ---------------------- | ------------------ |
| Screenshot from 2022-07-19 21-05-28 | Screenshot from 2022-07-19 21-04-22 |

File: index.html [hunk 2]
```diff
@@ -18,13 +17,17 @@
...
+                    <ul class="menu_bar">
+                        <li class="nav-item"><a href="#Works_Section" class="nav-link">Portfolio</a></li>
+                        <li class="nav-item"><a href="#About_Myself" class="nav-link">About</a></li>
+                        <li class="nav-item"><a href="#Contact_form" class="nav-link">Contact</a></li>
                     </ul>
+                    <div class="hamburger">
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
```
Comment (tresorsawasawa) [inline index.html:30]: - [ ] Also make sure to add the correct spacing around the `Hamburger`(top ** right) button to stick to the Figma design. For the `Top Spacing` I suggest you add to the whole top of the **Header**  then the right of the `Hamburger` button to stick to the Figma design.

| ### Figma DEsign | Yout Design |
| ------------------------- | ------------------------ |
| Screenshot from 2022-07-19 21-21-28 | Screenshot from 2022-07-19 21-22-50 |

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @RodChak :wave: 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @RodChak @mullenlucas 👋, 

## Status: Approved 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Great job implementing the changes suggested by the previous reviewer. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**